### ISSUE TITLE
Contact

### ISSUE THREAD

**Issue by jleclanche** [Luashine/wc3-debug-console-paste-helper#3]
Hi! Sorry for the random issue but I can't find any email or any way to contact you on your profile. I saw your issue in keg - can I talk to you somehow? Email's on my profile

**Comment by Luashine** [comment]
Sorry I've been busy this week and didn't notice the email. Do you want to talk in private?

**Comment by jleclanche** [comment]
Yeah :) Shoot me an email or some other form of contact: https://leclan.ch
